### ISSUE TITLE
[3.6.0] File Meta not working

### ISSUE THREAD

**Issue by hashandsaltthemes** [getkirby/kirby#3546]
**Describe the bug**  
The 3.6.0 alpha.1 now wont find file meta templates set up to work from `/site/blueprints/files/shot.yml` it gives the error `This file has no blueprint yet. You can define the setup in /site/blueprints/File.yml` when previous versions used /site/blueprints/files/default.yml

**To Reproduce**  
1. Set files to use a custom blueprint and place the blue print in `blurprints\files\myfiles.yml`
2. Upload a file to something using that blueprint (im using a files section)
3. Click on the image to edit its meta
4. You should see the message `This file has no blueprint yet. You can define the setup in /site/blueprints/File.yml`

**Expected behavior**  
Should use the meta file `blurprints\files\myfiles.yml` to edit the files meta data

**Additional context**  
I have a project in a private repo that i can give access too if it helps to resolve the issue.

**Comment by distantnative** [comment]
Hmmm I tried to reproduce this but works perfectly fine.
How is your blueprint setup for the files section you upload to?

**Comment by hashandsaltthemes** [comment]
Using images and videos as virtual pages.

Page blueprint (its a tab):

```
label: Client
icon: image

columns:
  # Content
  left:
    width: 1/3
    sections:
      content:
        type: fields
        fields:
          pagetext:
            label: Description
            type: textarea
            maxlength: 600
            buttons:
              - italic
              - bold
              - link
              - email

  bottom:
    width: 2/3
    sections:
      images:
        headline: Shots
        type: files
        layout: cards
        size: small

        limit: 30
        image:
          cover: true
          back: black
          ratio: 16/9
```
File blueprint:

```
title:               Shot

columns:
  left:
    width:           1/3
    sections:
      # Content
      meta:
        type:        fields
        fields:
          alt:
            label:   Title
            type:    text

          seotags:
            label:   Keywords
            type:    tags

          desc:
            label:   Description
            type:    textarea
            buttons: false
          photographer:
            type: users
            multiple: false

  focus:
    width:           2/3
    sections:
      # Content
      content:
        type:        fields
        fields:
          focus:
            label:   Focal Point
            type:    focus
```
Because im using images as virtual pages, i have a create hook that sets the template in the files meta:

```
    'hooks' => [
        'file.create:after' => function ($file) {
            if ($file->parent()->template() == 'client') {

                if ($file->type() == 'video') {                
                    $file->update([
                        'template'    => 'video'
                    ]);

                } elseif ($file->type() == 'image') {

                    $file->update([
                        'template'    => 'shot'
                    ]);
                }
            }
        }

  ],
```

**Comment by hashandsaltthemes** [comment]
Think i have just realised the issue. There was a typo in the hook that the sets the template.

However there is still the issue of the incorrect message if the blueprint cannot be found which is this:

```
This file has no blueprint yet. You can define the setup in /site/blueprints/File.yml
```

That differs from earlier versions and should probably read

```
This file has no blueprint yet. You can define the setup in /site/blueprints/files/default.yml
```

**Comment by lukasbestle** [comment]
The wrong error message should be fixed in 3.6.0-alpha.2. Could you please test again with alpha 2 which blueprint the Panel expects and can't find?

**Comment by hashandsaltthemes** [comment]
Tested with 3.6.0-alpha.2, everything working and message is now correct. :)
